lib/Twig/Cache/Null.php

```diff
@@ -52,7 +44,7 @@ public function load($key)
      */
     public function getTimestamp($key)
     {
-        // never called as has() always returns false
+        // never called as load() always returns false
```

Comment:
Right. The comment is obsolete anyway, because with `has()` being removed, we need to check the timestamp before `load()`.